packages/test/test-driver-definitions/package.json

```diff
@@ -82,13 +82,6 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {
-			"TypeAliasDeclaration_DriverEndpoint": {
-				"backCompat": false
-			},
-			"TypeAliasDeclaration_RouterliciousEndpoint": {
-				"backCompat": false
-			}
-		}
+		"broken": {}
```

Comment:
This change seems like it shouldn't be here? We only clean up this section as part of the release process.

Reply:
Hey Alejandro! I was just reverting the changes I made in the parent PR. I added this in when I added in frsCanary to RouterliciousEndpoint; https://github.com/microsoft/FluidFramework/pull/15054/files. 
we were removing the frsCanary option from RouterliciousEndpoint which causes build errors unless I remove this.